extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -576,6 +644,55 @@ describes.realWin(
           expect(postPromptUI).to.have.display('none');
         });
 
+        describe('hide/show postPromptUI with local storage', () => {
+          beforeEach(() => {
+            toggleExperiment(win, 'amp-consent-geo-override');
+            defaultConfig = dict({
+              'consentInstanceId': 'ABC',
+              'consentRequired': true,
+              'postPromptUI': 'test2',
+            });
+            consentElement = createConsentElement(doc, defaultConfig);
+            postPromptUI = doc.createElement('div');
+            postPromptUI.setAttribute('id', 'test2');
+            consentElement.appendChild(postPromptUI);
+            doc.body.appendChild(consentElement);
+            ampConsent = new AmpConsent(consentElement);
+          });
+
+          it('hide postPromptUI', async () => {
+            await ampConsent.buildCallback();
+            ampConsent.element.classList.remove('i-amphtml-notbuilt');
+            await macroTask();
+
+            expect(postPromptUI).to.not.be.null;
+            expect(postPromptUI).to.have.display('none');
+          });
+
+          it('shows postPromptUI with local storage decision', async () => {
+            const scheduleDisplaySpy = env.sandbox.spy(
```

Comment:
anyway to avoid stubbing this private method?

Reply:
Seems like stubbing it was overkill. Removed.